api/src/api/notes.js

```diff
@@ -45,6 +45,29 @@ router.get(
   }),
 );
 
+//GET /notes/labels
```

Comment:
This is super nitpicky 😅 but putting a space after // helps with consistency like the other comments